Source/Core/Core/HW/Memmap.cpp

```diff
@@ -149,9 +153,21 @@ bool IsInitialized()
 static MemoryView views[] =
 {
 	{&m_pRAM,      0x00000000, RAM_SIZE,      0},
+	{nullptr,      0x02000000, RAM_SIZE,      MV_MIRROR_PREVIOUS},
+	{nullptr,      0x04000000, RAM_SIZE,      MV_MIRROR_PREVIOUS},
+	{nullptr,      0x06000000, RAM_SIZE,      MV_MIRROR_PREVIOUS},
 	{nullptr,      0x200000000, RAM_SIZE,     MV_MIRROR_PREVIOUS},
+	{nullptr,      0x202000000, RAM_SIZE,     MV_MIRROR_PREVIOUS},
+	{nullptr,      0x204000000, RAM_SIZE,     MV_MIRROR_PREVIOUS},
+	{nullptr,      0x206000000, RAM_SIZE,     MV_MIRROR_PREVIOUS},
 	{nullptr,      0x280000000, RAM_SIZE,     MV_MIRROR_PREVIOUS},
+	{nullptr,      0x282000000, RAM_SIZE,     MV_MIRROR_PREVIOUS},
+	{nullptr,      0x284000000, RAM_SIZE,     MV_MIRROR_PREVIOUS},
+	{nullptr,      0x286000000, RAM_SIZE,     MV_MIRROR_PREVIOUS},
 	{nullptr,      0x2C0000000, RAM_SIZE,     MV_MIRROR_PREVIOUS},
+	{nullptr,      0x2C2000000, RAM_SIZE,     MV_MIRROR_PREVIOUS},
+	{nullptr,      0x2C4000000, RAM_SIZE,     MV_MIRROR_PREVIOUS},
+	{nullptr,      0x2C6000000, RAM_SIZE,     MV_MIRROR_PREVIOUS},
```

Comment:
I'm not sure I understand how exactly this makes locking more complicated... assuming we have mirrors at all, it's the same amount of work to implement 2 mirrors or 11.  But I can handle this differently if you think limiting the number of mirrors is important for locking.